src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  margin: 0;
```

Comment:
it's better to reset such things for the list of tags you really need
not critical, so I'll approve since all other comments are fixed